pkg/scheduler/framework/plugins/volumezone/volume_zone.go

```diff
@@ -242,51 +356,135 @@ func (pl *VolumeZone) Filter(ctx context.Context, cs *framework.CycleState, pod
 			v, ok = node.Labels[translateToGALabel(pvTopology.key)]
 		}
 		if !ok || !pvTopology.values.Has(v) {
-			logger.V(10).Info("Won't schedule pod onto node due to volume (mismatch on label key)", "pod", klog.KObj(pod), "node", klog.KObj(node), "PV", klog.KRef("", pvTopology.pvName), "PVLabelKey", pvTopology.key)
-			return framework.NewStatus(framework.UnschedulableAndUnresolvable, ErrReasonConflict)
+			logger.V(10).Info("Cannot schedule pod onto node due to volume (mismatch on label key)", "pod", klog.KObj(pod), "node", klog.KObj(node), "PV", klog.KRef("", pvTopology.pvName), "PVLabelKey", pvTopology.key)
+			return framework.NewStatus(framework.UnschedulableAndUnresolvable, ErrReasonConflict), false
 		}
 	}
-
-	return nil
+	return nil, true
 }
 
-func getStateData(cs *framework.CycleState) (*stateData, error) {
-	state, err := cs.Read(preFilterStateKey)
+// isSchedulableAfterPersistentVolumeClaimChange is invoked whenever a PersistentVolumeClaim added or updated.
+// It checks whether the change of PVC has made a previously unschedulable pod schedulable.
```

Comment:
Add a comment that a PVC becoming bound or using a WaitForFirstConsumer storageclass can cause it to become schedulable.